clusterloader2/pkg/state/namespaces_state.go

```diff
@@ -34,28 +34,28 @@ type InstancesState struct {
 type InstancesIdentifier struct {
 	Basename   string
 	ObjectKind string
-	ApiGroup   string
+	APIGroup   string
 }
 
 // namespaceState represents state of a single namespace.
 type namespaceState map[InstancesIdentifier]*InstancesState
 
-// namespacesState represents state of all used namespaces.
-type namespacesState struct {
+// NamespacesState represents state of all used namespaces.
+type NamespacesState struct {
```

Comment:
please don't expose it - it wasn't exposed before

Reply:
My concern is if I don't expose them, we'll get error when running `verify-golint.sh` (unless flag `-min_confidence=0.9` is used).

I tried to look if it's possible to disable golint on file `clusterloader2/pkg/state/state.go` but looks like that's not possible,

Thoughts?

Reply:
OK - I missed the PR description. It makes sense for now - we can try figuring out if we can somehow hide it later.